biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
So this means that for `case REMOVED` we want `content = Delta.MAJOR`? Which is what it was before this patch.  I am confused.